### DISCUSSION TITLE
Multiple Databases Management

### DISCUSSION THREAD

**Discussion by jorgebarrero** [directus/directus#16072] [Feature Requests]
As you know, Directus manages one single database at the time. I have just developed a system with multiple databases with Directus, but user management is hard specially if you have common users in all systems. SSO may be a solution but it requires additional effort to accomplish a single issue.

It would be nice if Directus could handle a shared database with all users, and share those users with other databases. The idea is just to keep users in one database (for authentication), and all other stuff on each database.

Or there is another simple solution that could be implemented?

**Comment by rijkvanzanten** [comment]
> Or there is another simple solution that could be implemented?

I don't think there's an easy way to just get that rolling without some heavy engineering effort

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
